Test failure due to missing test.dtb file in package distribution

When testing the zig-devicetree library after installation via Zig package manager, the tests fail because the `test.dtb` file referenced in `src/shared.zig` is not included in the distributed package.

```sh
$ zig test ~/.cache/zig/p/devicetree-3.0.0-cZRcrBZfAQCtpfpqy62PYa5QI1XU9mNuBgpTnI0r7MZz/src/DeviceTree.zig
/root/.cache/zig/p/devicetree-3.0.0-cZRcrBZfAQCtpfpqy62PYa5QI1XU9mNuBgpTnI0r7MZz/src/shared.zig:13:30: error: unable to open 'test.dtb': FileNotFound
    const emded = @embedFile("test.dtb");
                             ^~~~~~~~~~
referenced by:
    refAllDeclsRecursive [inlined]: /root/lang/zig/ziglang/lib/std/testing.zig:1188:21
    comptime: /root/.cache/zig/p/devicetree-3.0.0-cZRcrBZfAQCtpfpqy62PYa5QI1XU9mNuBgpTnI0r7MZz/src/shared.zig:28:37
    1 reference(s) hidden; use '-freference-trace=3' to see all references
```

Expected Behavior:​​
The test should run successfully.

--- a/build.zig.zon
+++ b/build.zig.zon
@@ -4,25 +4,11 @@
 .{
     .name = .devicetree,
 
-    .version = "3.0.0",
+    .version = "3.0.1",
 
-    .minimum_zig_version = "0.14.0",
+    .minimum_zig_version = "0.14.1",
 
-    .paths = .{
-        "LICENSE",
-        "LICENSES",
-        "REUSE.toml",
-
-        "build.zig",
-        "build.zig.zon",
-
-        "src/DeviceTree.zig",
-        "src/Node.zig",
-        "src/PHandle.zig",
-        "src/Property.zig",
-        "src/shared.zig",
-        "src/Tag.zig",
-    },
+    .paths = .{""},
 
     .fingerprint = 0x10ab9dbfac5c9471,
 }

